database/rrdcalc.h

```diff
@@ -251,8 +251,8 @@ void rrdcalc_rrdhost_index_init(RRDHOST *host);
 void rrdcalc_rrdhost_index_destroy(RRDHOST *host);
 
 #define RRDCALC_VAR_MAX 100
-#define RRDCALC_VAR_FAMILY "$family"
-#define RRDCALC_VAR_LABEL "$label:"
+#define RRDCALC_VAR_FAMILY "${family}"
```

Comment:
This change is not mentioned in the OP.

Reply:
Yeah, sorry, yes, that's changed as well. Although we don't use them ourselves in our stock alerts any more, I don't know if we should remove it.